sql/pgsql/tables/Media.sql

```diff
@@ -8,6 +8,7 @@ create table Media (
 	filename          varchar(255),
 	original_filename varchar(255),
 
+	has_hls boolean not null default false,
```

Comment:
We need to remember to add this sql to all site's databases when this is merged.